airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -573,6 +573,16 @@ def build_pod_request_obj(self, context=None):
         )
         return pod
 
+    def dry_run(self) -> None:
+        """
+        Prints out the pod definition that would be created by this operator
+        Does not include labels specific to the task instance (since there isn't
+        one in a dry_run) and excludes all empty elements.
+        """
+        print("dry run:")
```

Comment:
Would remove this line, will result in errors when people copy-paste the output and don't know about this one line.

Reply:
ok sure...